### PR TITLE
Extend conformances Photos, StoreKit, UIKit

### PR DESCRIPTION
**Pull request by larryonoff**

### CODE REVIEW

**Comment by stephencelis** [comment]
@larryonoff Thanks for the PR! We've noticed that some of these availability checks can become a bit precarious depending on the platform and release mode of the build, so we want to get some better CI into place before accepting more PRs like this, just to ensure they don't break anything. I'm looking into that soon, and once that's ready we will happily merge!

**Comment by larryonoff** [comment]
@stephencelis thanks for such a great library!

File: Sources/CustomDump/Conformances/UIKit.swift [hunk 1]
```diff
@@ -25,4 +25,23 @@
...
+      case .never:
+        return "UIScrollView.ContentInsetAdjustmentBehavior.never"
+      case .scrollableAxes:
+        return "UIScrollView.ContentInsetAdjustmentBehavior.scrollableAxes"
+      @unknown default:
+        return "UIScrollView.ContentInsetAdjustmentBehavior.(@unknown default, rawValue: \(self.rawValue))"
+      }
+    }
+  }
+}
```
Comment (stephencelis) [inline Sources/CustomDump/Conformances/UIKit.swift:46]: Looks like an extra brace snuck in.
```suggestion
```

**Comment by stephencelis** [comment]
Thanks!

**Pull request merged**